Rideau/RideauViewDragGestureRecognizer.swift

```diff
@@ -37,10 +37,16 @@ final class RideauViewDragGestureRecognizer: UIPanGestureRecognizer {
     trackingScrollView = event.findVerticalScrollView()
     super.touchesBegan(touches, with: event)
   }
-
+  
   override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent) {
-
     super.touchesMoved(touches, with: event)
+    
+    if state == .began {
+      let vel = velocity(in: view)
+      if abs(vel.x) > abs(vel.y) {
```

Comment:
I'm not sure what the right approach is here either, but wouldn't this be too sensitive to slanted drags? I'm worried that the gesture would fail a lot when a downward swipe that starts from a diagonal movement would cancel panning prematurely.